cabal-testsuite/PackageTests/Check/PackageFiles/VersionBounds/cabal.out

```diff
@@ -0,0 +1,11 @@
+# cabal check
+These warnings may cause trouble when distributing the package:
+Warning: [less-than-equals-upper-bounds] On library, these packages have less than or equals (<=) upper bounds:
+  - base
+Please use less than (<) for upper bounds. There is more information at https://pvp.haskell.org/
+Warning: [trailing-zero-upper-bounds] On library, these packages have upper bounds with trailing zeros:
+  - base
+Please avoid trailing zeros for upper bounds. There is more information at https://pvp.haskell.org/
+Warning: [greater-than-lower-bounds] On library, these packages have greater than (>) lower bounds:
+  - base
+Please use greater than or equals (>=) for lower bounds. There is more information at https://pvp.haskell.org/
```

Comment:
The message is very good, but does PVP actually explains clearly why we would need such and such bounds? Maybe it is/was/will be in the FVP FAQ?

Reply:
@andreasabel I found the repository behind https://pvp.haskell.org/ and was suprised to see it not listed as a [subsite of www.haskell.org](https://github.com/haskell-infra/www.haskell.org?tab=readme-ov-file#subsites). Can you link or quote the part of the PVP specification that motivates #9806?

Reply:
The PVP specification also doesn't mention `^>=`.